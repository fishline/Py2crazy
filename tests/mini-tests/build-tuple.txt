str = '%d %s %s' % (self.x, self.y, self.z)
