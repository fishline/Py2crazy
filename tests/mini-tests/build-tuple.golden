Disassembling top-level module in ./mini-tests/build-tuple.txt
  1       0 LOAD_CONST           str = [43m[30m'%d %s %s'[39m[49m % (self.x, self.y, self.z)
          3 LOAD_NAME            str = '%d %s %s' % ([43m[30mself[39m[49m.x, self.y, self.z)
          6 LOAD_ATTR            str = '%d %s %s' % ([43m[30mself.x[39m[49m, self.y, self.z)
          9 LOAD_NAME            str = '%d %s %s' % (self.x, [43m[30mself[39m[49m.y, self.z)
         12 LOAD_ATTR            str = '%d %s %s' % (self.x, [43m[30mself.y[39m[49m, self.z)
         15 LOAD_NAME            str = '%d %s %s' % (self.x, self.y, [43m[30mself[39m[49m.z)
         18 LOAD_ATTR            str = '%d %s %s' % (self.x, self.y, [43m[30mself.z[39m[49m)
         21 BUILD_TUPLE          str = '%d %s %s' % (self.x, self.y, [43m[30mself.z[39m[49m)
         24 BINARY_MODULO        str = [43m[30m'%d %s %s' % (self.x, self.y, self.z)[39m[49m
         25 STORE_NAME           [43m[30mstr[39m[49m = '%d %s %s' % (self.x, self.y, self.z)
         28 LOAD_CONST           [43m[30mstr[39m[49m = '%d %s %s' % (self.x, self.y, self.z)
         31 RETURN_VALUE         [43m[30mstr[39m[49m = '%d %s %s' % (self.x, self.y, self.z)
